lua/tardis/screens/sh_fallbackscreen.lua

```diff
@@ -34,32 +58,48 @@ TARDIS:AddScreen("New features fallback", {menu=false}, function(self,ext,int,fr
     power:SetFont("TARDIS-Default")
     power.DoClick = function()
         TARDIS:Control("power")
+    end    
+    
+    local repair=vgui.Create("DButton",frame)
+    repair:SetSize( frame:GetWide()*0.2, frame:GetTall()*0.2 )
+    repair:SetPos(frame:GetWide()*0.35 - repair:GetWide()*0.5,frame:GetTall()*0.15 - repair:GetTall()*0.5)
+    repair:SetText("Repair TARDIS")
+    repair:SetFont("TARDIS-Default")
+    repair.DoClick = function()
+        TARDIS:Control("repair")
     end
 
     local fastremat=vgui.Create("DButton",frame)
     fastremat:SetSize( frame:GetWide()*0.2, frame:GetTall()*0.2 )
     fastremat:SetPos(frame:GetWide()*0.13 - fastremat:GetWide()*0.5,frame:GetTall()*0.4 - fastremat:GetTall()*0.5)
-    fastremat:SetText("Toggle Fast Remat")
+    fastremat:SetText("Fast Remat PLH")
     fastremat:SetFont("TARDIS-Default")
     fastremat.DoClick = function()
         TARDIS:Control("fastremat")
     end
+    function fastremat:Think()
+        local on = ext:GetData("demat-fast")
```

Comment:
Setting text constantly may be computationally expensive, maybe check the existing text first or set another variable which corresponds with what is set for this control

Reply:
Alternatively have a look at the internal code of SetText if you can find it and see if a similar optimisation is already in place there

Reply:
Oh nvm I see you have fixed this anyway!